plugininfo.h

```diff
@@ -0,0 +1,100 @@
+/* This file is generated by the nymea build system. Any changes to this file will *
+ * be lost. If you want to change this file, edit the plugin's json file.          */
+
+#ifndef PLUGININFO_H
```

Comment:
You shouldn't commit this file. it'll be autogenerated on every build anyways.

Reply:
Sorry, my mistake. I actually know that it's generated upon build. I guess the best fix for this would be to add it to .gitignore?

Reply:
The thing with gitignore is that it breaks `git clean`...

Reply:
Also, the best fix is to do out-of source builds :)